examples/test/qore/debug/test-debug.qtest

```diff
@@ -70,17 +70,18 @@ class MyDebugProgram inherits DebugProgram {
         return actions[0];
     }
   }
-  handle(int func, ProgramControl pgm, reference runState, *int statementId, *hash extra) {
+  handle(int func, ProgramControl pgm, reference runState, reference runToStatementId, *int statementId, *hash extra) {
     *hash location;
     handleCount.inc();
     if (statementId)
       location = pgm.getStatementIdInfo(statementId);
-    #printf("%y: pgm: %y, %y, %y, %y\n", func, pgm, runState, location, extra);
+    #printf("%y: pgm: %y, %y, %y, %y, %y\n", func, pgm, runState, runToStatementId, location, extra);
     hash it = (
       'func': stResolve{func},
       'file': location.file,
       'line': location.start_line,
       'tid': gettid(),
+      'runToStatementId': runToStatementId,
     );
     any act = getAction(func);
```

Comment:
please use `auto` instead of `any` as `any` is subject to type stripping (performance issue)